config/rbac/role.yaml

```diff
@@ -4,48 +4,12 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
-- apiGroups:
```

Comment:
what are those changes? they are not described in the commit message.

Reply:
These changes were created automatically when the permissions were updated in the `notificationservice_controller.go` file.
Anyway, I reverted them.